wwwroot/index.html

```diff
@@ -1,198 +1,385 @@
+                </a>
+            </li>
+        </ul>
+        <section class="video-section">
+            <div class="video-homepage"> <img class="mobile_video_bg d-block" src="img/video_bg.png" alt="Wasabi Wallet - video"> </div>
             <div class="container">
-                <div class="row align-items-center position-relative">
-                    <div class="col-lg-12">
-                        <div class="text-center">
-                            <div class="epic-text title-lined mb-5"><strong>Privacy is</strong> your <strong>ability</strong> to <strong>selectively reveal</strong> yourself to the world</div>
-                        </div>
-                        <div class="row">
-                            <div class="col-lg-8 offset-lg-2">
-                                <p class="mb-5">
-                                    We live in an Orwellian surveillance society where your information is being used to typecast and manipulate you. Bitcoin projects are being pressured to collect more and more data, if possible.
-                                    <br />This is why Wasabi Wallet is programmed to be a zero-knowledge software. Developers can't collect any sensitive information about you. What you do with your bitcoin is your business.
-                                </p>
+                <div class="row vh-100">
+                    <div class="col-lg-11 offset-lg-1 position-relative">
+                        <div class="floating-home">
+                            <div class="row">
+                                <div class="col-lg-5">
+                                    <h1 class="title"><strong>Wasabi Wallet</strong></h1>
+                                    <p class="mb-4">The most popular Bitcoin wallet for desktop with coinjoin. Free & open-source. Non-custodial.
+                                        <br>Reclaim your privacy today.</p><a href="index.html#download" class="btn btn-primary mt-2 mt-md-0">Download</a> <a href="#bitcoinisnotprivate" class="btn btn-blur affix-arrowright mt-2 mt-md-0">Read more</a> </div>
+                                <div class="col-lg-4 offset-lg-3">
+                                    <div class="box-blur d-none d-lg-block">
+                                        <div class="title-small mb-3">No default coordinator</div>
+                                        <p>Starting from June 1, 2024, Wasabi Wallet will no longer have a default coinjoin coordinator.</p><strong><a href="https://blog.wasabiwallet.io/zksnacks-is-discontinuing-its-coinjoin-coordination-service-1st-of-june/" target="_blank" class="affix-arrowright">Don't trust, verify</a></strong> </div>
```

Comment:
the link to the shutdown announcement being "don't trust, verify" is weird
better rephrase it